script/test

```diff
@@ -54,6 +54,7 @@ install_test() {
 
 [ -z "$HUB_COVERAGE" ] || script/coverage prepare
 script/build
+make man-pages
```

Comment:
I don't think we strictly need man pages to be built during tests. We can work around it.